### ISSUE TITLE
Additional learning items

### ISSUE THREAD

**Issue by cscairns** [18F/devops-workshop#1]
Should consider adding:
- automated security scans
- automated code quality scans

**Comment by robertsosinski** [comment]
Good point.  Not sure what some good automated security scans are; any ideas?

There are a few ways to look at code quality.  Good news is that I would bundle all of these (testing, security, code quality, etc...) under the automated build process.

**Comment by cscairns** [comment]
some tools are framework-specific such as brakeman for rails/ruby...how about nessus?
